app/policies/user_policy.rb

```diff
@@ -0,0 +1,5 @@
+class UserPolicy < ApplicationPolicy
```

Comment:
Why do you need another Policy class? Shouldn't it be better to use the RentPolicy?